AlicerceTests/Sources/Network/MappableTestCase.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  MappableTestCase.swift
+//  Alicerce
+//
+//  Created by Luís Afonso on 07/04/2017.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import XCTest
+
+@testable import Alicerce
+
+final class MappableTestCase: XCTestCase {
+
+    // MARK: - Success tests
+
+    func testModel_WhenInputObjectIsValid_ItShouldReturnAFilledModel() {
+        let aValidDict = [
+            "data" : "👍"
+        ]
+
+        do {
+            let mappedModel = try MappableModel.model(from: aValidDict)
+
+            XCTAssertEqual(mappedModel, MappableModel(data: "👍"), "✅ it created a valid model")
+        } catch {
+            XCTFail("🔥 It thrown an invalid error 👉 \(error) 😱")
```

Comment:
Grammar 👮 : "An invalid error was thrown"

Also, I would call it "unexpected" instead of "invalid" 😝